File: check_htaccess.py
import ftplib
from io import BytesIO

def check_htaccess():
    try:
        ftp = ftplib.FTP('82.198.228.182')
        ftp.login('u754458241.morsall.com', 'l$9Qs3i]g0y]/V~k')
        ftp.cwd('/public_html')
        buf = BytesIO()
        ftp.retrbinary('RETR .htaccess', buf.write)
        print(".htaccess content:\n" + buf.getvalue().decode())
    except Exception as e:
        print(f"Error: {e}")

check_htaccess()

File: check_htaccess.php
<?php
$ht = '/home/u754458241/domains/morsall.com/public_html/.htaccess';
if (file_exists($ht)) {
    echo "<pre>";
    echo htmlspecialchars(file_get_contents($ht));
    echo "</pre>";
} else {
    echo ".htaccess NOT FOUND at $ht";
}
?>
